src/tests/Common/testenvironment.proj

```diff
@@ -172,17 +172,7 @@
     <TestEnvironment Include="jitstress_isas_2_x86_nosse41" JitStress="2" EnableSSE41="0" /> <!-- Depends on SSSE3 and SSE3_4 -->
     <TestEnvironment Include="jitstress_isas_2_x86_nosse42" JitStress="2" EnableSSE42="0" /> <!-- Depends on SSE41 -->
     <TestEnvironment Include="jitstress_isas_2_x86_nossse3" JitStress="2" EnableSSSE3="0" /> <!-- Depends on SSE3 -->
-
-    <!-- Trigger Sve code paths-->
-    <TestEnvironment Include="default_arm64_sve" AltJitName="clrjit_universal_arm64_x64.dll" AltJit="*" MaxVectorTBitWidth="128" />
-    <TestEnvironment Include="jitstress1_arm64_sve" JitStress="1" AltJitName="clrjit_universal_arm64_x64.dll" AltJit="*" MaxVectorTBitWidth="128" />
-    <TestEnvironment Include="jitstress2_arm64_sve" JitStress="2" AltJitName="clrjit_universal_arm64_x64.dll" AltJit="*" MaxVectorTBitWidth="128" />
-    <TestEnvironment Include="jitstressregs1_arm64_sve" JitStressRegs="1" AltJitName="clrjit_universal_arm64_x64.dll" AltJit="*" MaxVectorTBitWidth="128" />
-    <TestEnvironment Include="jitstressregs2_arm64_sve" JitStressRegs="2" AltJitName="clrjit_universal_arm64_x64.dll" AltJit="*" MaxVectorTBitWidth="128" />
-    <TestEnvironment Include="jitstressregs3_arm64_sve" JitStressRegs="3" AltJitName="clrjit_universal_arm64_x64.dll" AltJit="*" MaxVectorTBitWidth="128" />
-    <TestEnvironment Include="jitstressregs4_arm64_sve" JitStressRegs="4" AltJitName="clrjit_universal_arm64_x64.dll" AltJit="*" MaxVectorTBitWidth="128" />
-    <TestEnvironment Include="jitstressregs8_arm64_sve" JitStressRegs="8" AltJitName="clrjit_universal_arm64_x64.dll" AltJit="*" MaxVectorTBitWidth="128" />
-
+    <TestEnvironment Include="jitstress_isas_arm64_sve" AltJitName="clrjit_universal_arm64_x64.dll" AltJit="*" MaxVectorTBitWidth="128" /> <!-- Trigger Sve code paths-->
```

Comment:
Will this cause problems on an Arm64 box where I believe it would be `clrjit_universal_arm64_arm64.dll` instead?

Reply:
I'm guessing "no" by default since its set to run only on Windows_x64 machines?

Reply:
that's right...and that was the reason i did not have on linux/x64 either, otherwise would have to specify `libclrjit_universal_arm64_x64.so`